src/tile/tile.scss

```diff
@@ -175,9 +175,7 @@
       }
 
       > .iui-name-label {
-        text-overflow: ellipsis;
-        overflow: hidden;
-        white-space: nowrap;
+        @include iui-line-clamp(1);
```

Comment:
Can `iui-line-clamp` be used here? 
![image](https://user-images.githubusercontent.com/36186912/115185209-10d23300-a0e8-11eb-8b96-7da200bd7746.png)

Reply:
Done